homeassistant/components/samsungtv/bridge.py

```diff
@@ -234,6 +234,8 @@ def _get_remote(self, avoid_open: bool = False):
             except AccessDenied:
                 self._notify_callback()
                 raise
+            except (ConnectionClosed, OSError):
+                self._remote = None
```

Comment:
`self._remote` is already None?

Reply:
Yup. Will make it a `pass` statement.

Not used to side effect in init :)